test/http/billing_service.js

```diff
@@ -0,0 +1,70 @@
+var uuid      = require('uuid')
+var Promise   = require('bluebird')
+var codius    = require(__dirname+'/../../')
+var assert    = require('assert')
+var supertest = require('supertest')
+var server    = require(__dirname+'/../../billing-service/server')(codius)
+var http      = supertest(server)
+
+describe('Billing Service HTTP Interface', function() {
+  var token
+
+  before(function(done) {
+    new codius.Token({ token: uuid.v4() }).save()
+      .then(function(model) {
+        token = model
+        done()
+      })
+  })
+
```

Comment:
Only if you aren't cleaning out your test database after each run. We would also have to delete the associated `balance`, and and `credit` and `debit` records